### PR TITLE
LPS-187578 | Add Autom. Test FocusWillRemainOnCurrentToggleSetting

### PR DESCRIPTION
**Pull request by eltonccdantas**
[Ticket](https://liferay.atlassian.net/browse/LPS-187578)

Given Accessibility Menu
When toggle a configuration with "Enter" key
Then focus stays on the current configuration toggle

### CODE REVIEW

**Comment by eltonccdantas** [comment]
ci:test:accessibility-menu:nocompile

**Comment by eltonccdantas** [comment]
ci:test:sf

File: modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase [hunk 1]
```diff
@@ -213,15 +213,22 @@ definition {
 	@ignore = "Test Stub"
```
Comment (john-co) [inline modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase:213]: remove the ignore property

File: modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase [hunk 2]
```diff
@@ -213,15 +213,22 @@ definition {
...
 	@priority = 3
 	test FocusWillRemainOnCurrentToggleSetting {
+		task ("Given Accessibility Menu") {
```
Comment (john-co) [inline modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase:216]: add portal.acceptance = true

**Review by john-co — APPROVED** [review]
Please make changes and good to forward. Thanks!

**Comment by eltonccdantas** [comment]
ci:forward

**Pull request closed without merging**